ReconBuilderTest.py

```diff
@@ -0,0 +1,139 @@
+import ReconBuilder
```

Comment:
FYI, I think we are planning to use `unittest` library for unit testing. So when we move this code to empress we will change the format of this to use `unittest`. You might want to look up how to use the library.

Reply:
Could you convert the tests in this file to use `unittest`?